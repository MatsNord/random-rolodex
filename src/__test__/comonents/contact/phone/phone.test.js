import React from 'react';
import renderer from 'react-test-renderer';
import Phone from './../../../../components/contact/phone/phone';

test('PhoneItem render', () => {

  const tree = renderer.create(<Phone />).toJSON();
  expect(tree).toMatchSnapshot();
});