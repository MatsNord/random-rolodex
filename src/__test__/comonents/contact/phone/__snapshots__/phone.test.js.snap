// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneItem render 1`] = `
<div
  className="phone"
/>
`;
